ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_bgp.py

```diff
@@ -170,6 +172,41 @@ def router_bgp(self) -> dict | None:
             if neighbors:
                 router_bgp["neighbors"] = neighbors
 
+        for interface in self.shared_utils.wan_bgp_interfaces:
+            neighbor = {
+                "ip_address": get(interface, "peer_ip"),
+                "remote_as": get(interface, "bgp.peer_as"),
+                "peer_group": self.shared_utils.bgp_peer_groups["wan_underlay_peers"]["name"],
+            }
+
+            description = self.shared_utils.interface_descriptions.underlay_ethernet_interface(
+                InterfaceDescriptionData(
+                    shared_utils=self.shared_utils,
+                    peer=interface.get("peer"),
+                    peer_interface=interface.get("peer_interface"),
+                )
+            )
+            if description:
+                neighbor["description"] = description
+
+            neighbors.append(neighbor)
```

Comment:
The neighbors list is created in some `else` condition above, so may not be available here.
```suggestion
            router_bgp.setdefault("neighbors", []).append(neighbor)
```